src/app.ts

```diff
@@ -0,0 +1,93 @@
+import express, { Application, Request, Response, NextFunction } from "express";
+import * as bodyParser from "body-parser";
+import { check, validationResult } from "express-validator";
+
+const app: Application = express();
```

Comment:
it's already inference type, I'd like
```ts
const app = express()
``` 
will be easier to look

Reply:
fixed on 5e1eb54c01ef4a6b08ebc2d1c553572e5fa17e1a